core/chains/evm/config/toml/defaults/Blast_Mainnet.toml

```diff
@@ -0,0 +1,20 @@
+ChainID = '81457'
+FinalityDepth = 200
+FinalityTagEnabled = true
+ChainType = 'optimismBedrock'
+
+[GasEstimator]
+EIP1559DynamicFees = true
+BumpThreshold = 60
+BumpPercent = 20
```

Comment:
BumpPercent is already 20% https://github.com/smartcontractkit/chainlink/blob/develop/core/chains/evm/config/toml/defaults/fallback.toml#L42